src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -265,6 +268,14 @@ enum class AspectRatioMeetOrSlice
     Slice
 };
 
+enum class GaussianBlurEdgeMode
+{
+    None = 0,
+    Duplicate,
+    Wrap,
+    Mirror
```

Comment:
Is there a mirror mode? It's not in the spec.
https://www.w3.org/TR/SVG11/filters.html#feConvolveMatrixElementEdgeModeAttribute

Reply:
latest draft: https://drafts.fxtf.org/filter-effects/#element-attrdef-feconvolvematrix-edgemode
but I can remove since we don;t support this in the renderer